### PR TITLE
Update bike lane per capita with 2023 year data

### PR DESCRIPTION
**Pull request by elvbom**
Update bike lane per capita with 2023 year data (from Trafikverket and SMHI) and add test

### CODE REVIEW

**Comment by elvbom** [comment]
This is blocked by #583 since the SMHI data updates that are partially reflected here, due to the bike lane data being updated to showcase how the 2023 entry appears on the site.

PLEASE NOTE: the data pipeline must be rerun after the 2022 data entry from SMHI has been merged in.

File: data/climate_data_calculations.py
Comment (Greenheart) [inline data/climate_data_calculations.py:12]: We need to use the new import `calculate_bike_lane_per_capita` instead of `bicycle_calculations`

**Review by Greenheart — CHANGES_REQUESTED** [review]

File: data/climate_data_calculations.py
Comment (elvbom) [reply data/climate_data_calculations.py:12]: Resolved merge conflict gone wrong! I fix!

File: data/tests/test_bike_lane_per_capita.py
Comment (Greenheart) [inline data/tests/test_bike_lane_per_capita.py:1]: Very nice to have a test for this, to at least show the intention and what we need to look for :)

**Review by Greenheart — APPROVED** [review]
Great work! Works well, and several nice code improvements

**Pull request merged**
